_episodes/06-branching.md

```diff
@@ -116,13 +122,21 @@ nothing to commit, working tree clean
 ~~~
 {: .bash}
 
+>Tip: If need to, you can rename a branch using the `git branch -m` (move) command. The 
+syntax is >`git branch -m old-name new-name`.
+>
+>~~~
+>$ git branch -m title-cads title-caps
```

Comment:
I like this better; less chance of half of the students renaming their branch, and half of those giving it a new name they made up.